src/info/prob_specs_exercises.json

```diff
@@ -1,6 +1,6 @@
 {
-  "last_updated": "2022-01-17T05:00:00Z",
-  "problem_specifications_commit_ref": "574c14b34021cddca5e34f53c1326fafb80d28d8",
+  "last_updated": "2022-04-12T08:44:56Z",
```

Comment:
Yes, this key should probably now be named `problem_specifications_commit_time` instead.